sycl/include/sycl/detail/compression.hpp

```diff
@@ -0,0 +1,136 @@
+//==---------- compression.hpp --------------------------------------------===//
```

Comment:
Please, move this header to source directory.

Reply:
Fixed in 312bd3889ebc2487693e9e9b256ade01330e20b0